include/boost/math/special_functions/gegenbauer.hpp

```diff
@@ -0,0 +1,41 @@
+//  (C) Copyright Nick Thompson 2019.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_MATH_SPECIAL_CARDINAL_B_SPLINE_HPP
```

Comment:
The include guard name needs changing - cut and paste error?

Reply:
Yup, just got that one.